csaf_2.1/prose/edit/src/introduction-02-terminology-glossary.md

```diff
@@ -21,189 +21,189 @@ a specific version of a file in a version control system; a database table acces
 CSAF Library with Extended Validation
-:    A CSAF library that also satisfies the conformance target "CSAF extended validator".
+:    A CSAF Library that also satisfies the conformance target "CSAF Extended Validator".
 
 CSAF Library with Full validation
-:    A CSAF library that also satisfies the conformance target "CSAF full validator".
+:    A CSAF Library that also satisfies the conformance target "CSAF Full Validator".
 
 CSAF Management System
-:    program that is able to manage CSAF documents and is able to display their details as required by CSAF viewer.
+:    program that is able to manage CSAF Documents and is able to display their details as required by CSAF Viewer.
 
 CSAF Modifier
-:    CSAF post-processor which takes a CSAF document as input and modifies the structure or values of properties.
-The output is a valid CSAF document.
+:    CSAF Post-Processor which takes a CSAF Document as input and modifies the structure or values of properties.
+The output is a valid CSAF Document.
 
 CSAF Post-Processor
-:    CSAF producer that transforms an existing CSAF document into a new CSAF document,
+:    CSAF Producer that transforms an existing CSAF Document into a new CSAF Document,
 for example, by removing or redacting elements according to sharing policies.
 
 CSAF SBOM Matching System
-:    A program that connects to or is an SBOM database and is able to manage CSAF documents as
-required by CSAF management system as well as matching them to SBOM components of the SBOM database.
+:    A program that connects to or is an SBOM database and is able to manage CSAF Documents as
+required by CSAF Management System as well as matching them to SBOM components of the SBOM database.
 
 CSAF Producer
 :    program that emits output in the CSAF format
 
 CSAF Superseder
-:    A CSAF post-processor that transforms a given CSAF into a superseded one.
+:    A CSAF Post-Processor that transforms a given CSAF into a superseded one.
 
 CSAF Translator
-:    CSAF post-processor which takes a CSAF document as input and translates values of properties into another language.
-The output is a valid CSAF document.
+:    CSAF Post-Processor which takes a CSAF Document as input and translates values of properties into another language.
+The output is a valid CSAF Document.
 
 CSAF Viewer
-:    CSAF consumer that reads a CSAF document, displays a list of the results it contains,
+:    CSAF Consumer that reads a CSAF Document, displays a list of the results it contains,
 and allows an end user to view each result in the context of the artifact in which it occurs.
 
 CSAF Withdrawer
-:    A CSAF post-processor that transforms a given CSAF into a withdrawn one.
+:    A CSAF Post-Processor that transforms a given CSAF into a withdrawn one.
 
 CVRF CSAF Converter
-:    CSAF producer which takes a CVRF document as input and converts it into a valid CSAF document.
+:    CSAF Producer which takes a CVRF Document as input and converts it into a valid CSAF Document.
```

Comment:
Hmm - we didn't capitalize "CVRF Document" in the conformance.md => please unify (either way - your decision).

Reply:
went back to lower case document for CVRF document. Fixed.